eva_submission/eload_ingestion.py

```diff
@@ -0,0 +1,111 @@
+import string
+import subprocess
+
+from bs4 import BeautifulSoup
```

Comment:
Will the built-in lxml + xpath fit the bill for the requirements in this script? See [here](https://github.com/EBIvariation/eva-common-pyutils/blob/803b4140b9936df7e9973f05e0faadde1ad00c6d/ebi_eva_common_pyutils/config_utils.py#L40) for reference.

Reply:
lxml + xpath is fine, thanks